config/simu.yaml

```
@@ -17,11 +17,10 @@ use_gui: false
 # 当 segment_num = 1时，即用一段数据标定
 segment_num: 1
 selected_segment:
-  - {start_time: 0, end_time: 7, path_bag: /home/ha/catkin_oa_calib/src/OA-LICalib/data/bag/simu_bag.bag}
+  - {start_time: 0, end_time: 7, path_bag: /root/calib_ws/src/OA-LICalib/data/bag/simu_bag.bag}
```

Could you change this with

```suggestion
  - {start_time: 0, end_time: 7, path_bag: /root/catkin_oa_calib/src/OA-LICalib/data/bag/simu_bag.bag}
```

Also could you change the docker run command in README.md with this: 

```
"docker run -it --env="DISPLAY" --volume="$HOME/.Xauthority:/root/.Xauthority:rw" --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" --volume="$REPO_PATH:/root/catkin_oa_calib/src/OA-LICalib" calib:v1 bash"
```

Its look the volume path is wrong so if we change this, everything will okay.